data/en/2024/portuguese/agenda.yml

```diff
@@ -5,33 +5,94 @@ types:
-      color: "purple"
+      color: "blue"
+    - name: Jakarta EE & MicroProfile Sessions
+      id: session
+      color: "green"
     - name: Vendor Talks
       id: vendor
       color: "orange"
 
 items:
   - name: Welcome Session
     type: keynote
+    presenters:
+      - Maximillian Arruda
+      - Paulo Simoes
+    presenter: |
+      <li>Maximillian Arruda</li>
+      <li>Paulo Simoes</li>
+    time: 9:30 - 10:00 AM BRT (GMT-03:00)  
+  - name: Oracle - Vendor talk
+    type: vendor
     presenters:
       - Paulo Simoes
+    presenter: | 
+      <li>Paulo Simoes</li>
+    time: 10:00 - 10:30 AM BRT (GMT-03:00)
+    abstract: |
+      <p>TBD
+      </p>
+
+  - name: "Simplifying NoSQL Database Integration with Jakarta NoSQL: A Hands-on Approach"
+    type: session
+    presenters:
       - Maximillian Arruda
-    times: []
-  - name: 'Vendor talks'
-    presenters: [Oracle, IBM, Payara]
+    presenter: |
+      <li>Maximillian Arruda</li>
+    time: 10:30 - 11:15 AM BRT (GMT-03:00) 
+    abstract: |
+      <p>
+        Nowadays, it’s normal to see cloud-native solutions dealing with a huge amount of information. 
+        In this scenario, the NoSQL databases are getting a huge relevant position, but dealing with many 
+        NoSQL solutions from many different vendors is very difficult.
+      </p>
+      <p>
+       In terms of dealing with relational databases, there’s no doubt that JPA specification is a 
+       well-established API in the Java world, but what if there’s a well-defined way, pretty similar to JPA, to work with NoSQL with Java?
+      </p>
+      <p>
+       Jakarta NoSQL specification was created to increase the developer experience between Java and NoSQL databases. 
+       This spec allows developers to handle easily with many NoSQL databases.
+      </p>
+      <p>
+       In this talk, we will make a live coding session presenting a solution with Jakarta NoSQL and Jakarta Data. 
+       Even more, we’re going to mix other Jakarta EE specifications like Jakarta Faces and Jakarta WebSocket.
+      </p>
+  - name: "O compromisso da IBM com Java nativo na nuvem: Liberty, Jakarta EE e MicroProfile"
     type: vendor
+    presenters:
+      - Fabio Franco
+    presenter: | 
+      <li>Fabio Franco</li>
+    time: 11:15 - 11:45 AM BRT (GMT-03:00)
     abstract: |
-          <p>Oracle: Paulo Simoes</p>
-          <p>Payara: Marcelo Gonçalves</p> 
-          <p>IBM: Fabio Franco</p> 
+      <p>
+       É um momento emocionante para fazer parte da comunidade Java à medida que Jakarta EE e MicroProfile 
+       levam os padrões Java nativos na nuvem para o futuro. Nessa sessão, exploraremos o compromisso de 
+       longa data da IBM com o Java e sua evolução por meio dos grupos open-source, Jakarta EE e MicroProfile. 
+       Junte-se a nós para explorar a inovação que estamos promovendo por meio da implementação open-source 
+       Open Liberty e de nosso produto WebSphere Liberty.
+      </p>
+
+  - name: TDB
```

Comment:
I think you mean TBD here.

Are you okay with me merging these changes with the TBD session? Or should I hold off until we have the full agenda?

Reply:
Hi @oliviergoulet5! 

Yes! My typo... I mean TBD ... however, I'm still waiting for updates from vendor sessions but I think that's no problem to publish it right away... I'll be pushing the changes ASAP...

Reply:
I fixed it!